amazon/ionbenchmark/Command.py

```diff
@@ -1,8 +0,0 @@
-from enum import Enum
```

Comment:
QQ, do we still need COMMAND?

Reply:
This was an accidental deletion. I have re-added it.